sphinx/source/demo.rst

```diff
@@ -30,7 +30,7 @@ Given the above ``scenario.json`` this should create 2 nodes on the local machin
     :end-before: SNIPPET_END: create_network
     :dedent: 2
 
-Each transaction listed in the scenario is then sent to either the `primary` or a `follower` node. As the current implementation uses Raft for consensus, any ``Write`` transactions (which modify the KV) must be sent to the `primary`. Sending them to a `follower` will result in a ``TX_NOT_LEADER`` error response. ``Read`` transactions can be go to any node.
+Each transaction listed in the scenario is then sent to either the `primary` or a `backup` node. As the current implementation uses Raft for consensus, any ``Write`` transactions (which modify the KV) must be sent to the `primary`. Sending them to a `backup` will result in a ``TX_NOT_PRIMARY`` error response. ``Read`` transactions can be go to any node.
```

Comment:
Just noticed typo in last sentence: "can be go to any node", can you fix that too?